### DISCUSSION TITLE
New info with 2.4.0

### DISCUSSION THREAD

**Discussion by daeynasvistas** [ngraziano/LMICPP-Arduino#31] [General]
In the log i can see this now:

![image](https://user-images.githubusercontent.com/2634610/230198342-eff827f3-35f8-41be-855a-819385fbfee2.png)

```
114264920 RX CONTINUE
114264947 Decode Frame RXC
114264966 Invalid address
```

is this "normal"? some other sensor payload?

Using SX1276 and ESP32 pico with custom board.

thanks.

**Comment by ngraziano** [comment]
Hi,
I  think it is normal. It must be that the sensor receive a message for an other sensor and discard it.
